docs/usage/configuration-options.md

```diff
@@ -676,6 +676,11 @@ Example:
 
 ## gitLabAutomerge
 
+If you are using automerge you can use this setting to use the gitlab native automerge function for MR's to speedup automerge.
+Attention: This is a top-level key.
+To activate automerging within a `packageRule`, you must set `automerge` to `true` in the same `packageRule`.
```

Comment:
```suggestion
If you enabled automerge in the Renovate config, you can speed up the automerge process, by using GitLab's own automerge function.
Attention: This is a top-level key.
To activate automerging within a `packageRule`, you must set `automerge` to `true` in the same `packageRule`.
```

The proper noun GitLab should be capitalized like in my suggestion.
Also maybe the new text is clearer/easier to understand?